lib/pennyworth/commands/setup_command.rb

```diff
@@ -62,6 +62,20 @@ def read_os_release_file
       File.read(os_release_file) if File.exist?(os_release_file)
     end
 
+    def vagrant_installed?
+      @vagrant = Cheetah.run "rpm", "-q", "vagrant", stdout: :capture
+      @vagrant_version = @vagrant.lines.select{|plugin| plugin.start_with?("vagrant")}
+
+      !@vagrant.match(/vagrant-[1-]\.[7-]\.[2-]/).nil?
+    end
+
+    def vagrant_libvirt_installed?
+      @vagrant_libvirt = Cheetah.run "vagrant", "plugin", "list", stdout: :capture
+      @vagrant_libvirt_version = @vagrant_libvirt.lines.select{|plugin| plugin.start_with?("vagrant-libvirt")}
```

Comment:
Line is too long. [110/100]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.